ui/src/components/admin/AdminNav.js

```diff
@@ -1,24 +1,48 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Col, Row } from 'theme/system';
+import { Row } from 'theme/system';
+import styles from 'theme/adminNavStyles';
+import UserIcon from 'icons/UserIcon';
 
-export default () => (
-  <Col p={15}>
-    <Link to="/admin/manage-users">
-      <Row>
-        <Row p={10}>icon</Row>
-        <Col p={10}>
-          <Row>Manage Users</Row>
-        </Col>
-      </Row>
-    </Link>
-    <Link to="/admin/manage-models">
-      <Row>
-        <Row p={10}>icon</Row>
-        <Col p={10}>
-          <Row>Manage Models</Row>
-        </Col>
-      </Row>
-    </Link>
-  </Col>
+// Urls used in navigation
+const [base, manageUsersUrl, manageModelsUrl] = [
+  '/admin',
+  '/admin/manage-users',
+  '/admin/manage-models',
+];
+
+// Nav paths to handle "nested pages"
+const [modelsNavPaths, usersNavPaths] = [[base, manageUsersUrl], [manageModelsUrl]];
+
+// Nav active state func
+const navLinkClassname = (currentPath, navPaths) =>
+  navPaths.indexOf(currentPath.replace(/\/$/, '')) !== -1 ? 'nav-link active' : 'nav-link';
+
+export default ({ location: { pathname } }) => (
+  <Row className="admin-nav" css={styles}>
+    <div className="nav">
+      <Link className={navLinkClassname(pathname, modelsNavPaths)} to={manageUsersUrl}>
```

Comment:
https://reacttraining.com/react-router/web/api/NavLink/activeclassname-string

Reply:
Switched to styled components since than PR but yeah saw this as well, thanks dude

Reply:
haha I saw that you changed it like a minute after I posted that then I couldn't find my comment again to delete it. sorry!